eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -150,9 +150,10 @@ jobs:
           node common/scripts/install-run-rush.js install
         displayName: "Install library dependencies"
 
-      - script: |
+      - pwsh: |
```

Comment:
Just curious why the change from script to pwsh? While it should work I'm just curious if something triggered the change?

Reply:
This has been my recent discovery that when we write multiple commands, sometimes `script` executes the 2nd command before completing the first. pwsh doesn't do that. I was getting failures last week on another task bcz of using script. Then when I switched to pwsh and it worked as expected.

Reply:
Hmm.. that sounds odd I'd like to better understand that at some point. Script should definitely not be running the steps in parallel at the same time.